cli/src/commands/visualize/Types.d.ts

```diff
@@ -0,0 +1,61 @@
+export interface CALMInstantiation {
+    nodes: CALMNode[],
+    relationships: CALMRelationship[]
+}
+
+export type NodeType = 'actor' | 'system' | 'service' | 'database' | 'internal-network' | 'ldap' | 'dataclient';
+
+export interface CALMNode {
+    name: string, 
+    class?: string,
+    'unique-id': string,
+    'node-type': NodeType,
+    description: string,
+    'data-classification'?: string,
+    'run-as'?: string,
+    instance?: string
+}
+
+export type CALMRelationship = CALMInteractsRelationship | CALMConnectsRelationship | CALMDeployedInRelationship | CALMComposedOfRelationship;
+
+export interface CALMInteractsRelationship {
+    'relationship-type': {
```

Comment:
Is this top level property required in every relationship type?

Reply:
i.e. can we remove duplication

Reply:
hmmm, there is some commonality between them but I think because the ConnectsRelationship has some top level fields that set it apart from the others, it would be a little tricky to have a super type here